src/modules/Lan.rb

```diff
@@ -666,6 +666,8 @@ def Import(settings)
     # we export a 2-level map of typed "devices"
     # @return dumped settings
     def Export
+      return {} if NetworkService.is_network_manager
```

Comment:
I think this is wrong. I expect it will return `{ "managed" => true }` so it knows that it is NM.

Reply:
Yep, that is what should be expected when wicked is the default, but not otherwise.